tox.ini

```
@@ -1,16 +1,17 @@
 [tox]
-envlist = py38-django{32}
-skipsdist=true
+envlist = py38-django{32, 42}
+skipsdist = true
 
 [pytest]
-DJANGO_SETTINGS_MODULE = course_discovery.settings.test
+django_settings_module = course_discovery.settings.test
```

Reverting this back should resolve the bulk test failure across shards.
```suggestion
DJANGO_SETTINGS_MODULE = course_discovery.settings.test
```

---

Restored, thanks.